docs: update error recovery options table

jwwojak opened the pull request:
# Overview

Chapter 7, Software and Operations. Updates the table that lists error recovery options.

- Changes markdown table to HTML.
- No liquid detected row: adds 2 new recovery options
- Pipette overpressure row: removes nested unordered lists, not needed.

## Test Plan and Hands on Testing

Check the revised code to make sure it displays info in a formatted table.

Sandbox URL: see the table in the [error recovery section](http://sandbox.docs.s3-website.us-east-2.amazonaws.com/error-recovery-flex-manual/flex-manual/software-operation/#error-recovery).

## Changelog

See summary above.

## Review requests

Nothing specific.

## Risk assessment

Low, docs only.

jwwojak committed:
Update no liquid detected error options

- adds 2 new options to list

- change list from markdown to html

- remove nested unordered lists in pipette overpressure row.

- Adds a <br> tag to get uniform spacing between the first list phrase and bullet items, to match second list.

ecormany commented on docs/flex-manual/docs/software-operation.md:
```diff
@@ -282,11 +282,56 @@ Tap **Launch recovery mode** to see options for the particular type of error tha
+      <td>Occurs when a pipette encounters an empty well and expects a liquid to be present.</td>
+      <td>
+        <ul>
+          <li>Manually fill the empty well retry with the same tips.</li>
```
```suggestion
          <li>Manually fill the empty well and retry with the same tips.</li>
```

ecormany commented on docs/flex-manual/docs/software-operation.md:
```diff
@@ -282,11 +282,56 @@ Tap **Launch recovery mode** to see options for the particular type of error tha
+        <ul>
+          <li>Manually fill the empty well retry with the same tips.</li>
+          <li>Manually fill the empty well and retry with new tips.</li>
+          <li>Manually refill well and skip to the next step.</li>
```
For consistency:
```suggestion
          <li>Manually fill the empty well and skip to the next step.</li>
```

ecormany reviewed (commented):
One typo, one other small edit. Otherwise looks good!

jwwojak committed:
Apply suggestions from code review

Thank you for the review. Making changes now.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Review comment changes

- Commented out a section for images, missing image, lines 246-251. Not sure what to do. I think my other doc on LPC has an image that summarizes the LPC changes. Flagged for comment.

- Fixed table formatting, missing <td></td> tags

- Added lower case "note" in the admonition block. Works as "Note" but using lc for consistency.

Pls review. tks

jwwojak commented:
Found a few more things. Commented out a TK for a missing image. Not sure what to do. Please check it out. Lines 246-251.

ecormany commented on docs/flex-manual/docs/software-operation.md:
```diff
@@ -243,10 +243,12 @@ move the pipette quickly, but beware of crashing the pipette into
 TK image of 8.4 LPC
 <figcaption>Summary of new labware offsets ready to be applied to a protocol.</figcaption>
 </figure>
+-->
```
I think this is covered by the changes you made in #18900, so we can delete this comment and everything within it.

ecormany reviewed (approved):
Optionally remove HTML comment. Otherwise good to go!

jwwojak committed:
Removing commented code

This will be updated in a different PR.  #18900
